hcloud/actions/domain.py

```diff
@@ -5,6 +5,10 @@
 
 
 class Action(BaseDomain):
+    STATE_RUNNING = "running"
+    STATE_SUCCESS = "success"
+    STATE_ERROR = "error"
```

Comment:
Should be `STATUS_ERROR` to be consistent with naming in our API and other libraries.